src/dataflow/src/server.rs

```diff
@@ -157,8 +155,8 @@ pub enum SequencedCommand {
         /// The timestamp to advance to.
         advance_to: Timestamp,
     },
-    /// Request that feedback is streamed to the provided channel.
-    EnableFeedback(mpsc::UnboundedSender<WorkerFeedbackWithMeta>),
+    // /// Request that feedback is streamed to the provided channel.
```

Comment:
Remove these

Reply:
Will do. Do you think there are more? (( Did I do this, or a merge artifact? ))

Reply:
I assumed it was leftover from WIP that didn't get cleaned up. I didn't see anything else when I read through everything.

Reply:
👍